02-intermediate-ruby/exercises/arrays-of-objects.md

```diff
@@ -0,0 +1,83 @@
+# Exercise:  Processing an Array of Objects
+
+You've been contracted by a hip bar/grill in Singapore to finish writing a system calculating the bills for each order another developer started.  An `Order` will consist of a number of `OrderItem`s which will each have a name, quantity and unit price and indicate if alcohol is included.
```

Comment:
"finish developing a system that another developer started. This system should calculate a bill for the customer when they place their order."

Reply:
name, quantity, unit price and an indicator if alcohol is included.